crates/eth-rpc/Cargo.toml

```diff
@@ -40,6 +40,9 @@ tracing = "0.1.34"
 tracing-subscriber = { version = "0.3.3", features = ["env-filter"] }
 dojo-test-utils = { workspace = true }
 
+# for cross-compiling
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
This is needed so `openssl` is built from [source](https://docs.rs/openssl/latest/openssl/#vendored) for the right architecture